tests/robot-tests/tests/general_public/fast_track.robot

```diff
@@ -9,19 +9,11 @@ Test Setup          fail test fast if required
 
 
 *** Test Cases ***
-Navigate to publication release page
-    environment variable should be set    PUBLIC_URL
-    user navigates to public frontend    %{PUBLIC_URL}/find-statistics/pupil-absence-in-schools-in-england
-
-Click fast track link for 'Pupil absence rates' data block
-    user waits until h1 is visible    Pupil absence in schools in England
-    user opens accordion section    Pupil absence rates    id:content
-    user scrolls to accordion section content    Pupil absence rates    id:content
-    user waits until h3 is visible    Explore and edit this data online
-    user clicks link    Explore data
+Navigate to fast track link for 'Pupil absence rates' data block
+    user navigates to public frontend    %{PUBLIC_URL}/data-tables/fast-track/34beb07c-cac1-492b-8829-864abf8ac0e6
```

Comment:
Does this data block have the same id in all environments? It would be safer to click through to this from the button in the content rather than rely on the id.